### ISSUE TITLE
Queueing a single song doesn't work

### ISSUE THREAD

**Issue by solarnz** [tchaik/tchaik#3]
When I try to queue a single song, it results in a JavaScript error.

```
Uncaught TypeError: Cannot read property 'Groups' of undefined
```

**Comment by dhowden** [comment]
Yeah, sorry about that. The UI elements for adding individual tracks to playlists were left in to remind me to implement this!  The main problem is that playlist items are better suited to a grouping of tracks, so ideally if you add two tracks from the same source group (i.e. an Album) then the playlist would merge them into a grouping in the playlist.

**Comment by dhowden** [comment]
This was fixed in fee03e45
